tap_google_search_console/client.py

```diff
@@ -249,6 +268,12 @@ def get_access_token(self):
                           max_tries=2,  # Only retry once
                           interval=900,  # Backoff for 15 minutes in case of Quota Exceeded error
                           jitter=None)  # Interval value not consistent if jitter not None
+    # backoff for 5 times, with 10 seconds consistent interval
+    @backoff.on_exception(backoff.constant,
+                          Timeout,
+                          max_tries=5,
+                          interval=10,
+                          jitter=None)
```

Comment:
@harshpatel4crest Add a comment why we have added jitter

Reply:
Added comment